web/src/components/Instruction.js

```diff
@@ -11,29 +11,31 @@ HelpText.propTypes = {
   children: PropTypes.node.isRequired
 };
 
-const Instruction = ({ reverse, source }) => {
-  const heading = t([source, 'heading'], { defaultValue: false });
-  const short = t([source, 'short'], { defaultValue: false });
-  const detail = t([source, 'detail'], { defaultValue: false });
-  const helpText = t([source, 'helpText'], { defaultValue: false });
+const Instruction = ({ args, reverse, source }) => {
+  const heading = t([source, 'heading'], { defaultValue: false, ...args });
+  const short = t([source, 'short'], { defaultValue: false, ...args });
+  const detail = t([source, 'detail'], { defaultValue: false, ...args });
+  const helpText = t([source, 'helpText'], { defaultValue: false, ...args });
 
   return (
     <div>
       {heading && <h3>{heading}</h3>}
-      {reverse && detail && <p>{detail}</p>}
-      {short && <p className="strong">{short}</p>}
-      {!reverse && detail && <p>{detail}</p>}
+      {reverse && detail && <Md content={detail} wrapper="p" />}
```

Comment:
I'm curious about what this change gets us.

Reply:
Instead of making the `detail` just a `<p>` with no internal formatting, it renders it as Markdown wrapped in a `<p>`.